firebase-messaging/src/main/java/com/google/firebase/messaging/TopicsSubscriber.java

```diff
@@ -251,7 +251,8 @@ boolean performTopicOperation(TopicOperation topicOperation) throws IOException
     } catch (IOException e) {
       // Operation failed, retry failed only if errors from backend are server related error
       if (GmsRpc.ERROR_SERVICE_NOT_AVAILABLE.equals(e.getMessage())
-          || GmsRpc.ERROR_INTERNAL_SERVER_ERROR.equals(e.getMessage())) {
+          || GmsRpc.ERROR_INTERNAL_SERVER_ERROR.equals(e.getMessage())
+      || GmsRpc.TOO_MANY_SUBSCRIBERS.equals(e.getMessage())) {
```

Comment:
I'm not sure if it's just how GitHub is displaying it, but this probably should match the indentation of the previous line.

Reply:
Done